public/styles.css

```diff
@@ -758,7 +758,10 @@ td.right, th.right {
 td.row-id {
     font-weight: bold;
 }
-
+.row {
```

Comment:
I learnt we already have one. I only extend and modify the top and bottom margins. However, we add a new css class for that if necessary ?